### ISSUE TITLE
Empty app build fails on ios

### ISSUE THREAD

**Issue by damir** [rstoenescu/quasar-framework#10]
On android it works.

It fails on ios with blank screen and with this message:

_test[68142:1449022] Resetting plugins due to page load._

I then debugged js with this in script tag:
```javascript
      <script type="text/javascript">
      window.onerror = function (errorMsg, url, lineNumber) {
           alert('Error: ' + errorMsg + ' Script: ' + url + ' Line: ' + lineNumber);
      }
        @@appManifest
        quasar.boot(function() {
          quasar.start.app();
        });
      </script>
```
and got this alert:

Error: Error: Cannot load page. Script: undefined Line: 26

**Comment by rstoenescu** [comment]
Looking into it on Mondays.

**Comment by rstoenescu** [comment]
Can you tell me pls:
1. Xcode version are you using
2. Mac OS version
3. Phone OS version
4. Cordova version

Thanks!

**Comment by damir** [comment]
Here it is:
1. Xcode - 7.2
2. Mac OS - Yosemite 10.10.5
3. Phone OS - 9.2 from simulator
4. Cordova - 6.2.0

**Comment by damir** [comment]
Huh, the build is using cordova 4.1.1, this is log form xcode - Apache Cordova native platform version 4.1.1 is starting

cordova -v outputs 6.2.0, it was installed via npm.

**Comment by rstoenescu** [comment]
Delete the `wrapper` folder and `quasar wrap` it again.

It may be that you were using Cordova 4.1.1 when making the wrapper then you've upgraded Cordova to 6.2.0. Common issue when using Cordova. Are you in that situation as well?

Tell me if making wrapper again with the new version solves the issue. It may not, because I saw the same error reported for Cordova on Stackoverflow in many cases. Anyways, I'll get to the bottom of it.

**Comment by damir** [comment]
I removed cordova from the system via npm, wrap command complained with this which is ok:

You need Cordova globally installed (npm install -g cordova).

Then I installed it, 6.2.0 was downloaded but 'quasar wrap platform add ios' is somehow still using 4.1.1:

Adding ios project...
iOS project created with cordova-ios@4.1.1
Installing "cordova-plugin-crosswalk-webview" for ios
Discovered plugin "cordova-plugin-whitelist" in config.xml. Adding it to the project
Fetching plugin "cordova-plugin-whitelist@1" via npm
Installing "cordova-plugin-whitelist" for ios

How to resolve this?

Thank you.

**Comment by rstoenescu** [comment]
`quasar wrap` issues amongst other things `cordova create <folder>` command (in this case the folder is `wrapper`).

Try to create a Cordova project outside of your Quasar App folder (`cordova create test-folder`) and see if it installs cordova-ios 4.1.1. I would expect this to be the case, although latest cordova-ios version is [4.2.0](https://www.npmjs.com/package/cordova-ios). Trying to make sure Quasar CLI is not at fault here. Then we'll see what to do next.

**Comment by damir** [comment]
I am sorry, I misread the name, cordova-ios@4.1.1 is not same as cordova 6.2.0 :)

Yes, the result is the same when using cordova without quasar CLI:

Adding ios project...
iOS project created with cordova-ios@4.1.1
Discovered plugin "cordova-plugin-whitelist" in config.xml. Adding it to the project
Fetching plugin "cordova-plugin-whitelist@1" via npm
Installing "cordova-plugin-whitelist" for ios

**Comment by rstoenescu** [comment]
Currently installing OS X El Capitan on a virtualbox so I can dug in deeper. Can you tell me if running the default Cordova App (so the folder created earlier outside of Quasar folder) issues the same error when run in the simulator? Thanks!

**Comment by damir** [comment]
Default cordova app works.

**Comment by rstoenescu** [comment]
Reproduced the issue. Investigating and fixing.

**Comment by rstoenescu** [comment]
Fix will be available in v0.4.1

**Comment by rstoenescu** [comment]
Released v0.4.1 containing the fix.
